### DISCUSSION TITLE
How to pass extra params to the authConfig

### DISCUSSION THREAD

**Discussion by syntrydy** [soofstad/react-oauth2-pkce#35] [Q&A] [answered]
I wish to pass an extra parameter `&acr_values=github` to the auth url.

tried
```
  extraTokenParameters: {
    "acrValues": "github"
  },
```
but it's not working

**Comment by soofstad** [comment, marked as answer]
Hi, thank you for reporting this.

If it's indeed the auth request you want to add extra parameters to, you should put them in the 'extraAuthParameters' config option. 
Let me know if it's still not working 🙂

**Comment by syntrydy** [comment]
Hello @soofstad 

Thanks for the response. It works !!!

```
  extraAuthParameters: {
    "acrValues": "github",
  },
```
